css/index.css

```diff
@@ -1 +1,687 @@
+}
+
+/* HEADER */
+.row--header {
+  padding: 14rem 0 5rem;
+  border-bottom: 1px solid #000;
+}
+
+.col-50--header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.cta {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  width: 65%;
+  margin: 0 auto;
+  animation: fromleft 500ms ease-in-out;
+}
+
+.header__image-container {
+  text-align: center;
+  animation: fromright 500ms ease-in-out;
+}
+
+.cta__button {
+  border: 1px solid black;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.6rem;
+  padding: 5px 20px;
+  cursor: pointer;
+  transition: all 200ms;
+  width: 60%;
+}
+
+.cta__button:hover {
+  color: #000;
+  background: #dcf2fd;
+}
+
+/* IMAGES */
+.middle-img-container {
+  margin-bottom: 50px;
+  background-image: url('../img/mid-page-accent.jpg');
+  background-size: cover;
+  background-position: top;
+  width: 100%;
+  height: 18.2rem;
+}
+
+.middle-img {
+  width: 100%;
+}
+
+/* BOTTOM CONTENT */
+.bottom-content--row {
+  padding-bottom: 50px;
+  border-bottom: 1px solid black;
+}
+
+/* CONTACT */
+.contact {
+  font-size: 1.6rem;
+  flex-direction: column;
+}
+
+.contact a,
+.contact a:visited {
+  color: #000;
+  text-decoration: none;
+  transition: all 300ms;
+}
+
+.contact a:hover {
+  color: #ccc;
+}
+
+/* FOOTER */
+.footer {
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
+  padding-bottom: 2rem;
+}
+
+/* PAGE STYLES */
+
+.page,
+.main--index {
+  opacity: 0;
+  animation: fadein 500ms ease-in 200ms forwards;
+}
+
+/* PAGE HEADER */
+.header--page {
+  padding-top: 10rem;
+  flex-direction: column;
+  margin-bottom: 3rem;
+}
+
+.heading-primary--page {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 3.6rem;
+  text-transform: uppercase;
+}
+
+.paragraph--page {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.8rem;
+  line-height: 1.7;
+}
+
+.page h2 {
+  margin-top: 2rem;
+}
+
+.page p {
+  text-align: left;
+}
+
+/* FEATURES PAGE */
+
+.features i {
+  color: #cbecff;
+  /* color: #999; */
+}
+
+.features,
+.product {
+  text-align: center;
+}
+
+/* PRODUCT PAGE */
+
+.product .price {
+  color: #cbecff;
+  font-size: 5rem;
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+  margin-bottom: 1rem;
+}
+
+.product .price span {
+  font-size: 1.6rem;
+  color: #000;
+  margin-left: 0.5rem;
+}
+
+.product p {
+  margin-bottom: 2rem;
+}
+
+/* SERVICES */
+.services-img {
+  margin-bottom: 2rem;
+}
+
+.services .row:first-of-type {
+  padding-bottom: 2rem;
+  border-bottom: 1px solid black;
+}
+
+.services .row {
+  justify-content: space-between;
+}
+
+.services .row .col-50 {
+  width: 48%;
+}
+
+.services-bottom-img {
+  width: 100%;
+}
+
+.services .list {
+  margin-top: 2rem;
+  padding-left: 2.27%;
+}
+
+.services .list .list-item {
+  margin-bottom: 1rem;
+}
+
+/* CARD */
+
+.card {
+  background-color: #f4f3f4;
+  margin-bottom: 3rem;
+  padding: 2rem;
```

Comment:
when using rems in the box-model, make sure that you want these things to change when the default browser font changes. If you don't, use percentages instead